### PR TITLE
Update for LICENSE file

### PR DESCRIPTION
**Pull request by fernando-sendMail**
Hey there!
I noticed that the `LICENSE` file includes the instructions on how to apply the license to your project, since it already has all the terms and conditions listed out directly then its not necessary to have that piece at the end of the license, I removed unnecessary text on license file.

Please take a look.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -173,30 +173,3 @@
       defend, and hold each Contributor harmless for any liability
       incurred by, or claims asserted against, such Contributor by reason
       of your accepting any such warranty or additional liability.
-
-   END OF TERMS AND CONDITIONS
```
Comment (stweil) [inline LICENSE]: You removed two lines too much. Otherwise this looks reasonable.

**Comment by amitdo** [comment]
https://github.com/apache/spark/blob/master/LICENSE
https://github.com/apache/mesos/blob/master/LICENSE
https://github.com/google/flatbuffers/blob/master/LICENSE.txt
https://github.com/google/yapf/blob/master/LICENSE

**Comment by fernando-sendMail** [comment]
@amitdo Thanks for the reference, I didn't know most project just took all the text and had it on their licenses

**Comment by amitdo** [comment]
I don't know if it represents most organization.

My above samples are from Apache (the organization who publishes the Apache 2.0 license), and Google (Tesseract is written since 2006 mostly by Google employees).

Note that in the Google case I gave 3 **different** examples.

**Comment by fernando-sendMail** [comment]
@stweil sounds reasonable to me, thanks.

**Pull request closed without merging**